client-sdk/ts-web/playground/sample-envoy.yaml

```diff
@@ -0,0 +1,74 @@
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 42280
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          codec_type: AUTO
+          stat_prefix: ingress_http
+          route_config:
+            virtual_hosts:
+            - name: ts_dev_vh
+              domains: ['*']
+              routes:
+              - match:
+                  safe_regex:
+                    google_re2:
+                      max_program_size: 933
+                    # You'd normally not permit NodeController/RequestShutdown, for example.
+                    regex: '/oasis-core\.(Scheduler/(GetValidators|GetCommittees|StateToGenesis|WatchCommittees)|Registry/(GetEntity|GetEntities|GetNode|GetNodeByConsensusAddress|GetNodeStatus|GetNodes|GetRuntime|GetRuntimes|StateToGenesis|GetEvents|WatchEntities|WatchNodes|WatchNodeList|WatchRuntimes)|Staking/(TokenSymbol|TokenValueExponent|TotalSupply|CommonPool|LastBlockFees|Threshold|Addresses|Account|Delegations|DebondingDelegations|StateToGenesis|ConsensusParameters|GetEvents|WatchEvents)|KeyManager/(GetStatus|GetStatuses)|Storage/(SyncGet|SyncGetPrefixes|SyncIterate|Apply|ApplyBatch|GetCheckpoints|GetDiff|GetCheckpointChunk)|RuntimeClient/(SubmitTx|GetGenesisBlock|GetBlock|GetBlockByHash|GetTx|GetTxByBlockHash|GetTxs|QueryTx|QueryTxs|WaitBlockIndexed|WatchBlocks)|EnclaveRPC/(CallEnclave)|Consensus/(SubmitTx|StateToGenesis|EstimateGas|GetSignerNonce|GetEpoch|WaitEpoch|GetBlock|GetTransactions|GetTransactionsWithResults|GetUnconfirmedTransactions|GetGenesisDocument|GetStatus|WatchBlocks)|ConsensusLight/(GetLightBlock|GetParameters|StateSyncGet|StateSyncGetPrefixes|StateSyncIterate|SubmitTxNoWait|SubmitEvidence)|NodeController/(RequestShutdown|WaitSync|IsSynced|WaitReady|IsReady|UpgradeBinary|CancelUpgrade|GetStatus))'
```

Comment:
Ouch :facepalm:

Reply:
it was this or replicate the whole `{match: ..., route: ...}` for each method :facepalm: 
or perhaps per module, if we're willing to whitelist at that granularity